src/components/network/PageNetworks.vue

```diff
@@ -6,8 +6,8 @@
       <NetworkList :networks="mainNetworks" />
       <h3>Test Networks</h3>
       <NetworkList :networks="testNetworks" />
-      <h3>Coming soon...</h3>
-      <NetworkList :networks="[polkadot]" />
+      <h3>Coming Soon...</h3>
```

Comment:
```suggestion
      <h3>Coming Soon</h3>
```

Reply:
It is adding some thrill to it, don't you agree?